string component # component to act
string action # action to make

string CMD_INIT = init
string CMD_RELEASE = release
string CMD_GRIP = release
string CMD_STOP = stop